src/content-helper/dashboard-widget/top-posts/component-list.tsx

```diff
@@ -31,12 +30,7 @@ function TopPostList() {
 			provider.getTopPosts()
 				.then( ( result ): void => {
 					const mappedPosts: TopPostData[] = result.map(
-						( post: TopPostData ): TopPostData => (
-							{
-								...post,
-								date: getDateInUserLang( new Date( post.date ), SHORT_DATE_FORMAT ),
-							}
-						)
+						( post: TopPostData ): TopPostData => ( { ...post } )
```

Comment:
Is this mapping needed now? seems like `result` already have the required structure.

Reply:
Good catch!